### ISSUE TITLE
WWN Going Away...

### ISSUE THREAD

**Issue by rwrife** [tonesto7/nest-manager#656]
It appears that new developer accounts can no longer generate OAuth tokens for integrations now that the WWN program is going away and Google is closing the door to 3rd party integrations....has anyone else been able to generate a token?  I get 404 and 403 errors all over the Nest Developer site now.

**Comment by Alpalma** [comment]
Yes, the same here. Control with ST is gone after nest merge  with google @tonesto7 
I update my account and everything stopped. the app stays stuck trying to do the login. So I have to activated the new Alexa skill made by google.  Now I lost all the Nest manager automation from smart things what a shame... :(

**Comment by mvevitsis** [comment]
WWN is dead and you cannot make new connections. Someone will have to rewrite this using native APIs instead of WWN. Like this project here:
https://github.com/chrisjshull/homebridge-nest

**Comment by tonesto7** [comment]
I'm pretty sure that's pointless because the nest app is going away.

**Comment by mvevitsis** [comment]
Somehow the home bridge integration linked above works without WWN and can even be used with a newer google account.

**Comment by barabba9174** [comment]
This morning I was ready to migrate Nest Manager on Google Account, basing on homebridge-nest repo. 
The main issue is: I cannot save "cookies" as appSetting, there is an error 500 (maybe the string exceeds the length, it's a 1018 digits).
Any idea?

**Comment by clmcgrath** [comment]
i switched to an ecobee because of this :(
